installer/ServerInstaller.cs

```diff
@@ -50,7 +50,7 @@ public class ServerInstaller : MumbleInstall {
 		var licenseDir = new Dir("licenses");
 		var menuDir = new Dir(@"%ProgramMenu%");
 		var shortcutDir = new Dir("Mumble");
-		var menuShortcut = new ExeFileShortcut("Murmur", "[INSTALLDIR]murmur.exe", arguments: "");
+		var menuShortcut = new ExeFileShortcut("Mumble (server)", "[INSTALLDIR]mumble-server.exe", arguments: "");
```

Comment:
I think I would label it Mumble Server instead of putting server in parentheses.

_Mumble_ is the project or client. I would rather have Mumble Server always be labelled explicitly. That is the same with how we name the executable - mumble for the client and mumble-server for the server.

Reply:
I overtook the naming scheme from how the installer seems to be named (see further up in the same file)

Reply:
At first I had it in the same way as you suggest, so I will change that back...